src/stylesheets/app.scss

```diff
@@ -0,0 +1,298 @@
+@import "base/vars";
+@import "base/typography";
+
+body {
+  background-color: $florence-light-blue; // only temporary
+  margin: 0;
+}
+
+ul {
+  padding: 0;
+}
+
+sup {
+  color: #e95e67;
+  text-decoration: underline;
+}
```

Comment:
Maybe move `body`, `ul`, and `sup` to a separate sass file in base (or another directory) called **elements.scss**?